src/components/selector-dropdown/SelectorDropdown.js

```diff
@@ -241,9 +246,20 @@ class SelectorDropdown extends React.Component<Props, State> {
                         },
                         setActiveItemID,
                     };
+
                     if (index === activeItemIndex) {
                         itemProps.isActive = true;
                     }
+
+                    if (index === dividerIndex) {
```

Comment:
Since these two return statements are pretty similar, would it be cleaner to do something like the following to have a single return?
```
const hasDivider = index === dividerIndex;

return (
    <React.Fragment>
        hasDivider && <hr className="SelectorDropdown-divider" />
        {React.cloneElement(item, itemProps)}
    </React.Fragment>
);
```

Reply:
I agree that is cleaner. I'll update this.